helm/logging-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "resource.default.name" . }}
+  namespace: {{ include "resource.default.namespace" . }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      {{- include "labels.selector" . | nindent 6 }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      annotations:
+        releaseRevision: {{ .Release.Revision | quote }}
+      labels:
+        {{- include "labels.common" . | nindent 8 }}
+    spec:
+      volumes:
+      - name: {{ include "name" . }}-configmap
+        configMap:
+          name: {{ include "resource.default.name"  . }}
+          items:
+          - key: config.yml
+            path: config.yml
+      serviceAccountName: {{ include "resource.default.name"  . }}
+      securityContext:
+        runAsUser: {{ .Values.pod.user.id }}
```

Comment:
I'd rather we use security context like upstream is doing rather that the old giantswarm way like https://github.com/prometheus-community/helm-charts/blob/44df643f30e8e623f4bb95f646de2662522eb554/charts/kube-prometheus-stack/values.yaml#L756